code/modules/cards/singlecard.dm

```diff
@@ -24,7 +24,7 @@
 	/// The name of the card
 	var/cardname = "Ace of Spades"
 	/// Is the card flipped facedown (FALSE) or flipped faceup (TRUE)
-	var/flipped = FALSE
+	var/flipped = CARD_FACEDOWN
```

Comment:
these defines are not safe here, they're only working because of file names rn. need to be moved up to the __ level

Reply:
Or kill the procs :)